e2e/cypress/integration/scroll/fixed_width_spec.js

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
```

Comment:
Can you please remove the prod tag on the test. This is only added by the QA team once the test works fine continuously for 5 days.

Reply:
:heavy_check_mark: